jerry-debugger/jerry_client_main.py

```diff
@@ -885,13 +887,13 @@ def _parse_source(self, data):
                 return ""
 
             elif buffer_type in [JERRY_DEBUGGER_SOURCE_CODE, JERRY_DEBUGGER_SOURCE_CODE_END]:
-                source_code += data[1:]
+                source_code += data[1:].decode('latin1')
```

Comment:
Yes utf-8 should be used.

Originally I used `latin1` in some places because in some cases due to how the data was in the buffer, a `utf-8` byte sequence could be split apart, and the the first part of it could not be decoded, causing an error; but after further investigation decoding the bytes elsewhere, where their values are used eliminates this issue.